lib/surface_site_web/live/slots/binding_slots_to_generators.ex

```diff
@@ -58,7 +58,9 @@ defmodule SurfaceSiteWeb.GettingStarted.BindingSlotsToGenerators do
       <div>
         <Grid items={{ album <- @albums }}>
           <Column title="Title">
-            {{ album.name }} (Released: <strong>{{ album.released }}</strong>)
+            {{ album.name }}
+            (Released:
+            <strong>{{ album.released }}</strong>)
```

Comment:
There are multiple examples in this PR of newlines `\n` in potentially surprising places.

The rules at play here:

- Interpolations/HTML elements/text blocks are considered distinct children, and are by default put on separate lines.
- If a text node (such as the trailing `)` here) has no whitespace between its sibling, no whitespace will be added.

This is why we see the above behavior. Here are the nodes:

- `{{ album.name }}` (interpolation)
- `(Released: ` (text)
- `<strong>...</strong>` (HTML element)
- `)` (text)

I wouldn't be surprised if someone found this formatting odd. It would be possible to add "newline detection", so that if the original code has multiple nodes on one line, and they don't violate the line length, we keep them on one line.

Reply:
Yeah, that looks odd. We need to find a way to keep it in the same line.

Reply:
Sounds good; I'll do a little refactoring to make this possible.